tests/hpc/ganglia_server.pm

```diff
@@ -40,7 +41,13 @@ sub run {
 
     #install web frontend and start apache
     zypper_call('in ganglia-web');
-    assert_script_run('a2enmod php7');
+    # SLE15 has installed by default php7, SLE12 has php5
+    if (is_sle('15+')) {
```

Comment:
```
my $php_mod =  is_sle('15+') ? 'php7' : 'php5';
assert_script_run('a2enmod ' . $php_mod);
```
looks better as for me.
I see that you replace assert_script_run with script_run . Any good reason for that ?